client/src/pages/ProfileUser.js

```diff
@@ -1,8 +1,12 @@
 import React from "react";
-import { Link } from "react-router-dom";
+import { Link, Redirect } from "react-router-dom";
```

Comment:
If we have redirect here, we're going to need it in all the pages tied to once the user has signed in to double check they're authorized, so this logic will either have to go a could places or we put it higher up in the content.js file somehow to decide if those are accessible routes.